crates/aptos-genesis/src/config.rs

```diff
@@ -241,3 +240,49 @@ impl FromStr for HostAndPort {
         }
     }
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct OwnerConfiguration {
+    pub owner_account_address: AccountAddress,
+    pub owner_account_public_key: Ed25519PublicKey,
+    pub voter_account_address: AccountAddress,
+    pub voter_account_public_key: Ed25519PublicKey,
+    pub operator_account_address: AccountAddress,
+    pub operator_account_public_key: Ed25519PublicKey,
+    pub stake_amount: u64,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct OperatorConfiguration {
```

Comment:
should we have a pool_address field here, and assert it matches  owner_account_address (same as we do for operator_account_address)

Reply:
Agreed! would be nice to be able to verify the owner address here